applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -461,6 +486,103 @@ class MechanicalExplicitStrategy
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This method computes the explicit contribution of the constraints
+     * @param pScheme The pointer to the integration scheme used
+     * @param rModelPart The model part which defines the problem
+     */
+    void ComputeExplicitConstraintConstribution(
```

Comment:
I only have one request, this function has nothing to do with explicit, it fact it is also needed in many other places (e.g. the eigen-postprocessing also needs this)
=> please put it to an independent file in the core e.g. `ReconstructConstraintSlaveSolutionUtility` or so
(this way it could also be used from python, which might come in handy at some point)

FYI @adityaghantasala

Reply:
@philbucher @loumalouomega i really think that we need support for this from the MasterSlaveConstraints (the famous apply method...)

Reply:
I created the Apply method I will put it in this PR and in a standalone PR